pyproject.toml

```diff
@@ -16,57 +16,45 @@ exclude = '''
 line_length = 88
 profile = "black"
 
-
 [build-system]
-requires = ["setuptools>=61.2"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
+build-backend = "poetry_dynamic_versioning.backend"
 
-[project]
+[tool.poetry]
 name = "openshift-cluster-management-python-wrapper"
-authors = [
-    {name = "Meni Yakove", email = "myakove@gmail.com"},
-    {name = "Ruth Netser", email = "rnetser@gmail.com"}
-]
 description = "Wrapper around https://github.com/openshift/openshift-cluster-management-python-client"
+version = "0.0.0"
+authors = ["Meni Yakove <myakove@gmail.com>", "Ruth Netser <rnetser@gmail.com>"]
+readme = "README.md"
+homepage = "https://github.com/RedHatQE/openshift-cluster-management-python-wrapper"
+keywords = ["Openshift", "OCM"]
 classifiers = [
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent",
 ]
-requires-python = ">=3.8"
-keywords = ["Openshift", "OCM"]
-dependencies = [
-    "colorlog",
-    "openshift-cluster-management-python-client",
-    "openshift-python-wrapper",
-    "requests",
-    "pyyaml",
-    "openshift-python-utilities",
-    "rosa-python-client",
-    "python-simple-logger",
-    "redhat-qe-cloud-tools",
-    "importlib",
-    "python-benedict",
-]
-dynamic = ["version"]
-
-[tool.setuptools.dynamic]
-version = {file = "VERSION"}
-
-[project.readme]
-file = "README.md"
-content-type = "text/markdown"
-
-[project.urls]
+packages = [{include = "ocm_python_wrapper"}]
+# Include ocm-python-wrapper package data:
+include = [{path = "manifests/*.json"}]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+colorlog = "^6.7.0"
+python-benedict = "^0.32.0"
+python-simple-logger = "^1.0.5"
+openshift-cluster-management-python-client = "*"
```

Comment:
Please do not add any dependencies with `"*"`
remove those from the file and run:
`poetry add <package name>` to add them with versions